add basic errata-tool CLI

ktdreyer opened the pull request:
This installs a uility "errata-tool" in the user's $PATH.

yazug commented on errata_tool/cli/erratum.py:
```diff
@@ -0,0 +1,32 @@
+
+
+def get(argv):
+    if not argv:
```
want to use argparse? or something other than brute force argv processing?

yazug commented on errata_tool/cli/erratum.py:
```diff
@@ -0,0 +1,32 @@
+
+
+def main(argv):
+    if not argv or argv[0] == '--help' or argv[0] == '-h':
```
argparse whould help here

ktdreyer replied:
@yazug I spent a lot of time last week trying to do this entirely with argparse, and I went through a couple of iterations, but it's very difficult when you have subcommands that have their own help unique text with examples. For example argparse always wants to handle the "--help" flag in the main parser, and it never passes --help down to the subparsers.

In Ceph we use a full dispatcher https://github.com/alfredodeza/tambo for this type of problem, but I didn't want to add Tambo as a dependency just yet.

yazug replied:
did you get anywhere with this idea? otherwise we may just want to look at pulling this in as is and make incremental improvements over time.

ktdreyer replied:
I did not make time to rewrite it all with argparse. :cry: 

I do think that would be a valuable future effort that we could pursue after we merge this PR.

ktdreyer commented:
I've rewritten this to completely use argparse (no more direct `sys.argv` handling). I added a few new features to: you can now create advisories with this, or list all advisories for a given release.
